scripts/appS3.py

```diff
@@ -0,0 +1,95 @@
+import string
+import time
+import random
+import boto3
+import json
+import argparse
+
+
+def app_handle(args):
+    keys_path = args.keys
+    endpoint = args.endpoint
+
+    # Fetch Auth
+    with open(keys_path, 'r') as keys_file:
+        keys_dict = json.load(keys_file)
+
+    # Create Boto3 Client
+    keys = keys_dict["keys"][0]
+    client = boto3.resource("s3", verify=False,
+                            endpoint_url=endpoint,
+                            aws_access_key_id=keys["access_key"],
+                            aws_secret_access_key=keys["secret_key"])
+
+    # Perform IO
+    objects = []
+    bucket_name = "test-bucket"
+    client.Bucket(bucket_name).create()
+    for i in range(args.obj_num):
+        object_name = "test-object"+rand_str(4)
+        data = str(rand_str(random.randint(10, 30)))*1024*1024
+        primary_object_one = client.Object(
+            bucket_name,
+            object_name
+        )
+        primary_object_one.put(Body=data)
+        # Store for
+        objects.append(
+            (object_name, primary_object_one.content_length/(1024*1024))
+        )
+
+    # Print Summary
+    print("IO Summary: Object Count {}".format(args.obj_num))
+    for obj, size in objects:
+        print("Object: {}/{} -> Size: {}MB".format(bucket_name, obj, size))
+
+    # Cleanup (if asked for)
+    if not args.no_delete:
+        print("Performing Cleanup")
+        for obj, size in objects:
+            client.Object(bucket_name, obj).delete()
+
+
+def rand_str(length: int):
+    return "".join(
+        random.choices(string.ascii_uppercase + string.digits, k=length)
+    )
+
+
+if __name__ == "__main__":
+    # seed random
+    random.seed(time.time())
```

Comment:
This is probably not necessary. We don't need high level of randomness, and typically the random module would either use an OS-provided randomness source (e.g. /dev/urandom) or use system time automatically